content/_index.en.md

```diff
@@ -14,9 +14,13 @@ usecases:
     caption_1: uses OpenFisca and its
     caption_2: to inform Barcelona citizens on their eligibility to social benefits.
 
-  reforms:
+  leximpact:
     title: Evaluate reforms impact
     description: Compare the law now to what it could be.
+    caption_1: uses OpenFisca and its
+    caption_2: evaluate the ex ante impact of tax and benefit system reforms, thanks to a simulator dedicated to the income tax.
```

Comment:
```suggestion
    caption_2: to evaluate the ex ante impact of tax and benefit system reforms, thanks to a simulator dedicated to the income tax.
```

Reply:
Add, like in the French text, that it's a mean to quickly evaluate the impact... ?
> `thanks to a simulator` seems strange after reading the french text where lexImpact is the simulator.

Reply:
> Add, like in the French text, that it's a mean to quickly evaluate the impact... ?

I didn't add it as I didn't know how to say it naturally in English in a way that it preserves the same communicational intent.

Do you have an idea? For example:

> to quickly evaluate

would work for you?

I'm not an English-native so whatever seems good for you, I'm not picky.

> Thanks to a simulator seems strange after reading the french text where lexImpact is the simulator.

English and French texts are different, not a translation of one to the other.

It is a matter of choice: not being a translator myself, and given the adagio [Traduttore, traditore](https://fr.wikipedia.org/wiki/Traduttore,_traditore), I preferred to write both texts independently (although trying to stick to the same communicational intent).

Your note is then correct: the French text talks about LexImpact as a web application, the English one as a product (where the simulator is part of the product, not the product itself).

Maybe we could prefer one to the other to be consistent. If the choice was mine, I wouldn't, as the English text makes sense for a broader audience, to specify we're working with income tax and not the whole legislation.

On the other hand, after second read, I'm not quite satisfied with the French one, as it is too technical.

But I'm keen to modify the English one if you have a proposal.

What do you think?

Reply:
I proposed a new wording, let me know what do you think!